src/components/types.ts

```diff
@@ -43,12 +43,12 @@ export interface ComponentTag {
   order: number
 }
 type ComponentOptionValidator<T> = (value: T) => T | undefined | null
-/** 组件选项信息
- * @todo 需要 extends 出更具体的 Option 类型, 现在这样混一起太乱
- */
-export interface ComponentOption {
+export interface UnknownOptions {
+  [name: string]: unknown
```

Comment:
```suggestion
  // [name: string]: unknown
  [name: string]: any
```

初期可以先换回 `any`, 保证兼容性. 改造组件那边的类型的时候再临时切回 `unknown` 来查看 TS 的检查.